diagramming-tool/src/Component/Dashboard/ShapeTypes.js

```diff
@@ -14,4 +14,5 @@ const ShapeTypes = {
   };
   
   export default ShapeTypes;
-  
\ No newline at end of file
+  
+  // "test": "react-scripts test --watchAll --coverage --no-threads",`
```

Comment:
Please note and remove the commented lines in Next PR

Reply:
ok